apps/charterafrica/src/payload/blocks/Mooc.js

```diff
@@ -0,0 +1,81 @@
+import { validateHexColor, MuiButtonColors } from "../utils/colors";
+
+const { default: link } = require("../fields/link");
+
+const Mooc = {
+  slug: "mooc",
+  fields: [
+    {
+      name: "title",
+      type: "group",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      required: true,
+      localized: true,
+      fields: [
+        {
+          name: "color",
+          type: "text",
+          label: {
+            en: "Color",
+            fr: "Couleur",
+            pt: "Cor",
+          },
+          validate: validateHexColor,
+          required: true,
+        },
+        {
+          name: "content",
+          type: "richText",
+          required: true,
+          localized: true,
+          admin: {
+            elements: ["h1"],
+            leaves: ["bold", "italic", "underline", "code"],
+          },
+        },
+      ],
+    },
+    {
+      name: "link",
+      type: "group",
+      fields: [
+        {
+          name: "color",
+          type: "select",
+          options: MuiButtonColors,
+          required: true,
+          label: {
+            en: "Button Color",
+            fr: "Couleur du bouton",
+            pt: "Cor do botão",
+          },
+          admin: {
+            isClearable: true,
+            isSortable: true,
+          },
+        },
+        link({}),
+      ],
+    },
```

Comment:
```suggestion
    linkGroup(),
```

Just remember to import `linkGroup` in place of `link`.